card/lib/card.rb

```diff
@@ -36,6 +36,7 @@ class Card < ActiveRecord::Base
   after_save :extend
   
   TRACKED_FIELDS = %w(name type_id db_content trash)
-  Loader.load_mods if count > 0
 
+  ActiveSupport.run_load_hooks(:card, self)
```

Comment:
I think this is pretty important to the load/configure api.  We hook onto this in Cardio to load mods, but devs can also if need be.  This would make it easy to have several of those for loading mods in different places.  That might argue for configuring mods differently.  Maybe we don't need paths entries and just have the load call take an argument with the mod location.